skytemple/module/dungeon/controller/fixed.py

```diff
@@ -534,6 +568,13 @@ def _load_settings(self):
         self.builder.get_object('settings_override').set_active(self.override_id)
         self.builder.get_object('settings_width').set_text(str(self.floor.width))
         self.builder.get_object('settings_height').set_text(str(self.floor.height))
+        self.builder.get_object('settings_complete').set_active(bool(self.properties.null&0x1))
+        self.builder.get_object('settings_boss').set_active(bool(self.properties.null&0x2))
+        self.builder.get_object('settings_free').set_active(bool(self.properties.null&0x4))
```

Comment:
Maybe check if the patch isn't applied and if so use the fixed room ID to set the value of these (copying the hardcoded checks in the game)? That way people can see the hardcoded values instead of these being always 0 when the patch isn't applied.